mDNSShared/uds_daemon.c

```diff
@@ -2913,7 +2915,11 @@ mDNSlocal CFArrayRef _get_unicast_discovery_dns_services(request_state *info)
 mDNSlocal mStatus add_domain_to_browser(request_state *info, const domainname *d)
 {
     browser_t *b, *p;
+#if MDNSRESPONDER_PLATFORM_APPLE
     __block mStatus err;
+#else
+    mStatus err;
+#endif
```

Comment:
This has been upstreamed in 2200.0.8. Regardless, I don't think they take PRs in these repos.

Reply:
Has 2200.0.8 resolved all the issues in this PR? If not, I hope this PR can be merged into 1790. If this PR cannot be merged into 1790, how about resubmitting a PR to the master?

Reply:
Thank you for your response. I hope that the upstream provide a channel to accept patches. I am waiting for the upstream to fix the issues in this PR, and then I will create a new package in [Conan-Center](https://github.com/conan-io/conan-center-index/tree/master/recipes/mdnsresponder).